bindings/node/examples/documentation/quicktour.test.ts

```diff
@@ -1,182 +1,163 @@
 /* eslint-disable */
-var globRequire = require;
```

Comment:
I think it's best to keep semi colons performance wise

Reply:
And if not performance reasons, the [ASI](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#automatic_semicolon_insertion) can run into ambiguous cases, I personally prefer with than without :)

Reply:
Thanks for the heads up, the prettier setup complained, and the default linter from napi-rs said it should.

Reduces my trust in napi-rs by a solid amount if they're defaults aren't really good. (Also the GH action was insane)